style.css

```diff
@@ -68,46 +65,337 @@
     line-height: 24px;
     color: #344563;
     margin-bottom: 10px;
-    padding-left: 20px;
   }
 
   .social-media {
     display: flex;
     align-items: center;
-    gap: 10px;
+    gap: 18px;
+  }
+
+  .link-text a {
+    text-decoration: none;
   }
 
   .social-media li {
     list-style: none;
-    padding-left: 20px;
   }
 
   .social-media li a {
     text-decoration: none;
     color: #505f79;
   }
 
+  .language-set a img {
+    padding-left: 20px;
+    margin-top: 10px;
+  }
+
   .social-media li a img {
     width: 20px;
   }
 
-  .btn {
+  .link-text {
     font-size: 17px;
     font-weight: 500;
-    text-align: center;
-    padding: 15px;
     width: 200px;
     color: #172b4d;
     margin-bottom: 10px;
     cursor: pointer;
     transition: 0.3;
-    margin-left: 20px;
   }
 
-  .btn:hover {
-    background-color: #5e6c84;
+  /* Works Section */
+
+  .works-section {
```

Comment:
[OPTIONAL] Kindly fix the background color of the work section like on the Figma design.
<img width="607" alt="Screenshot 2023-01-13 at 9 00 35 PM" src="https://user-images.githubusercontent.com/25887253/212388844-e41e7f31-bc1d-4cb7-ab92-466d8f8c42fa.png">